.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=88
```

Comment:
Is there an open issue for project toml support?

Reply:
https://gitlab.com/pycqa/flake8/issues/428